### PR TITLE
Scrum rotation

### PR DESCRIPTION
**Pull request by HamishBrownPFR**
working on #7869

### CODE REVIEW

**Comment by HamishBrownPFR** [comment]
@hol353 please review and merge if happy

File: ApsimNG/Presenters/GraphPresenter.cs [hunk 1]
```diff
@@ -186,6 +186,13 @@ public void DrawGraph(IEnumerable<SeriesDefinition> definitions)
                         double yDouble = 0;
                         if (y is DateTime)
                             yDouble = ((DateTime)y).ToOADate();
+                        else if (y is String)
+                        {
+                            if ((String)y == "True")
+                                yDouble = 1.0;
+                            if ((String)y == "False")
+                                yDouble = 0.0;
```
Comment (hol353) [inline ApsimNG/Presenters/GraphPresenter.cs:194]: Why is this needed? Why not use a boolean?

Comment (HamishBrownPFR) [reply ApsimNG/Presenters/GraphPresenter.cs:194]: The graphs seem to work alright without this fix (ie this fix does not affect graphs appearance) but there is a non-fatal bug that keeps popping up in the debugger which is a bit annoying.  This change stops that bug

Comment (hol353) [reply ApsimNG/Presenters/GraphPresenter.cs:194]: I'd like to see that bug reproduced. Can you show me tomorrow via a Teams call? I don't really want this code in the graph presenter.

Comment (HamishBrownPFR) [reply ApsimNG/Presenters/GraphPresenter.cs:194]: I will pull the code change out of this pull request to keep this issue separate.  Will give you a call to show you the bug separately
